.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "kpsr-core"]
```

Comment:
Not acceptable. Use `find_package` instead

Reply:
Done.